openthread/Kconfig

```diff
@@ -84,6 +90,10 @@ config OPENTHREAD_NORDIC_LIBRARY_FTD
 	select OPENTHREAD_SLAAC
 	select OPENTHREAD_SNTP_CLIENT
 	select OPENTHREAD_UDP_FORWARD
+	select OPENTHREAD_DUA if OPENTHREAD_THREAD_VERSION_1_2
+	select OPENTHREAD_MLR if OPENTHREAD_THREAD_VERSION_1_2
+	select OPENTHREAD_LINK_METRICS_INITIATOR if OPENTHREAD_THREAD_VERSION_1_2
```

Comment:
No need for Link Metrics Initiator for non-battery powered devices.

Reply:
All in all You are correct but link metrics might be used also to decrease interference with other boards not only to reduce the battery consumption.